src/shared/components/cancerSummary/CancerSummary.tsx

```diff
@@ -0,0 +1,431 @@
+import * as React from "react";
+import * as _ from 'lodash';
+import {Checkbox} from 'react-bootstrap';
+import {computed, observable} from "mobx";
+import {observer} from "mobx-react";
+import Slider from 'react-rangeslider';
+import Select from 'react-select';
+import { MSKTabs, MSKTab } from "shared/components/MSKTabs/MSKTabs";
+import {If, Then, Else} from 'react-if';
+import {ThreeBounce} from 'better-react-spinkit';
+import {FormGroup, ControlLabel, FormControl} from 'react-bootstrap';
+import jsPDF from 'jspdf';
+
+import 'react-select/dist/react-select.css';
+import 'react-rangeslider/lib/index.css';
+
+import SummaryBarGraph from './SummaryBarGraph';
+
+export interface ICancerAlterationPlotData {
+    [alteration:string]:number;
+    mutated:number;
+    amplified:number;
+    deleted:number;
+    multiple:number;
+}
+
+export interface IBarGraphDatasets {
+    label: string;
+    totalCases: number;
+    altTotalPercent: number;
+    total: number;
+    count: number;
+    percent: number;
+    data: number[];
+    backgroundColor: string;
+}
+
+export interface IBarGraphData {
+    labels: string[];
+    datasets: IBarGraphDatasets[];
```

Comment:
are these labels different than the `label` property of `IBarGraphDatasets`?

Reply:
No. The `labels` array is just the full list of all the `label`s

Reply:
in my opinion that should be thrown out, if its accessible from the `IBarGraphDataset(s)` - better to have one source of truth